<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationPreviewer</class>
 <widget class="QDialog" name="AnimationPreviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview Animation</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="AnimatedLabel" name="animationLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QToolButton" name="playButton">
       <property name="toolTip">
        <string>Play</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons.qrc">
         <normaloff>:/Icons/Icons/play.png</normaloff>:/Icons/Icons/play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pauseButton">
       <property name="toolTip">
        <string>Pause</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons.qrc">
         <normaloff>:/Icons/Icons/pause.png</normaloff>:/Icons/Icons/pause.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="previousFrameButton">
       <property name="toolTip">
        <string>Previous Frame</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons.qrc">
         <normaloff>:/Icons/Icons/previousframe.png</normaloff>:/Icons/Icons/previousframe.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nextFrameButton">
       <property name="toolTip">
        <string>Next Frame</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons.qrc">
         <normaloff>:/Icons/Icons/nextframe.png</normaloff>:/Icons/Icons/nextframe.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Frame Delay:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="frameDelayInput">
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedLabel</class>
   <extends>QLabel</extends>
   <header>AnimatedLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
